zerver/tests/test_subs.py

```diff
@@ -612,6 +617,93 @@ def test_can_remove_subscribers_group_on_stream_creation(self) -> None:
             "'can_remove_subscribers_group' setting cannot be set to 'role:nobody' group.",
         )
 
+    def test_stream_topic_access_on_stream_creation(self) -> None:
```

Comment:
This test is almost same as ` test_can_remove_subscribers_group_on_stream_creation`. Is that okay?

Reply:
Yeah I can do that in follow up.